tests/prague/eip7692_eof_v1/eip3540_eof_v1/container.py

```diff
@@ -426,16 +421,6 @@
         ],
         validity_error=EOFException.MISSING_TERMINATOR,
     ),
-    Container(
-        name="multiple_code_and_data_sections_2",
-        sections=[
-            Section.Code(Op.STOP, code_outputs=NON_RETURNING_SECTION),
-            Section.Data(data="0xAA"),
```

Comment:
Here I'm unsure what is this multiple `Section.Data()` doing. Maybe better to just forbid declaring multiple?

Reply:
I think the original intention was to have `04 1234 04 2345` in the header or something along the lines. This one was designed back when EOF had a different header format iirc.

Reply:
Ok, make sense, but hard to hell if the `Container` is doing so. Probably not, because the tests `code-data-code-data` and code-code-data-data` generate the same bytecodes.